### ISSUE TITLE
Issues with mzXML from Bruker DataAnalysis and multiply charged species

### ISSUE THREAD

**Issue by JakeHaeckl** [mzmine/mzmine#2001]
### Basic information

* My operating system and version: Windows 10
* My mzmine version: 4.2

### What happened

We normally use a script to apply mass calibration and convert our Bruker data to mzXML but when I try to open it in mzmine, it says that the files are corrupted mzXML. After opening an example file, it seems like the format has changed a bit between what we get from msconvert (image left = DataAnalysis output, image right = MSconvert). 
![DataAnalysis_vs_MSconvert](https://github.com/user-attachments/assets/7ba0aca2-de31-4395-8670-88f9c3942ff7)

I also tried to directly import the .baf files using the "in subfolders" method of raw data import for the original Bruker files, but that did not work for me. 

Normally, I would simply proceed with the MSconvert version, but I am running into an issue with multiply charged species not associating the MS/MS spectrum with the lowest isotope and rather only the highest intensity isotope. I am not sure the the two issues are related, but I tried many different approaches with MSconvert and in and out of DataAnalysis to remedy this with no success.

The ions for Pinensin A I am interested in are 738.9975 at 5.45 which is the lowest m/z isotope of the [M+3H]3+ ion but the MS/MS is associated only to the 739.3318 ion. My assumption is this is simply user error at some point, but this is a scenario we will encounter quite a lot and I would like to determine a robust path forward for myself and the rest of our group.

The project can be found here:
https://drive.google.com/file/d/1H-IXa7b6XmUDhYkW89lFVeutwa_uSq6V/view?usp=sharing

Thanks for your time and help.

**Comment by SteffenHeu** [comment]
Hi, can you try to drag and drop the baf file into mzmine? The in sub folders option is sketchy for folder based file formats.

**Comment by JakeHaeckl** [comment]
I tried dragging and droping the original .d folder and it says that the task is finished, but no data is imported. This also happens if I drag the original analysis.baf file or attempt to do the same with any other original file within the .d 

![image](https://github.com/user-attachments/assets/597a9d96-f633-4cdf-9c9c-da64bd714f26)

**Comment by SteffenHeu** [comment]
Ok that's odd. can you please upload the .d file?

The mzxml issue is related to this one, i think: 
https://github.com/mzmine/mzmine/issues/1851

**Comment by JakeHaeckl** [comment]
https://drive.google.com/drive/folders/1Mwos2f5B1xSd_WBBdmdktvnPvqE4LpfO?usp=sharing

This should contain the project and the two .d files I attempted to work with in this case.

**Comment by SteffenHeu** [comment]
HI, i have no problem importing the files from the .d folder. What exactly did you do? Drag & drop the .d folder into the "MS data files" tab?
![grafik](https://github.com/user-attachments/assets/d38c31ac-c778-4bc9-a7a8-67010dc28edc)

**Comment by SteffenHeu** [comment]
If youre using a timsTOF i can only recommend to use timsControl instead of otof control, then you get the newer .tsf files.

**Comment by JakeHaeckl** [comment]
Yeah I did the same drag and drop into the ms files. This wouldn't be a 4.2 vs 4.1.7 issue? I'm the future we'll try to stick with timsControl but for our maXis4G data we'll have to stick with otofControl. I'll try to drag and drop again and see if it works.

**Comment by JakeHaeckl** [comment]
I tried dragging and dropping again into the MS data files tab and still get nothing. I will try to see if I can find a way forward. I will also play around with those settings you suggest for the ms/ms pairing. Thanks for your help and I'll let you know of any updates.
